src/AppBar/AppBar.js

```diff
@@ -27,23 +28,29 @@ export const styleSheet = createStyleSheet('AppBar', (theme) => {
   };
 });
 
-export default class AppBar extends Component {
-  static propTypes = {
-    accent: PropTypes.bool,
-    children: PropTypes.node,
-    className: PropTypes.string,
-    primary: PropTypes.bool,
-  };
+type DefaultProps = {
+  primary: boolean,
+};
 
-  static defaultProps = {
-    primary: true,
-  };
+type Props = {
+  accent?: boolean,
+  children?: Object|string,
+  className?: string,
+  primary?: boolean,
+};
 
+export default class AppBar extends Component<DefaultProps, Props, void> {
```

Comment:
I'm not sure what you are saying, or what specifically your links are pointing to.